<launch>
    <arg name="n_agents" />
    <arg name="map" />
    <arg name="command" />
    <rosparam file="$(find odrm_nav)/config/$(arg map)/poses$(arg n_agents).yaml" command="load" ns="poses" if="$(eval arg('map') == 'x')"/>
    <rosparam file="$(find odrm_nav)/config/$(arg map)/poses8.yaml" command="load" ns="poses" unless="$(eval arg('map') == 'x')"/>

    <node pkg="odrm_nav" type="goal_manager_benchmark.py" name="goal_manager_benchmark" output="screen">
			<param name="n_agents" value="$(arg n_agents)" />
			<param name="benchmark_data_folder" value="$(find odrm_nav)/res/" />
			<param name="command" value="$(arg command)" />
    </node>

</launch>